packages/cli-e2e/utils/platform.ts

```diff
@@ -4,6 +4,28 @@ import {HTTPRequest} from 'puppeteer';
 // TODO: CDX-98: URL should vary in function of the target environment.
 export const platformHost = 'https://platformdev.cloud.coveo.com/rest/';
 
+export async function listPlatformResource(
```

Comment:
Any reason we are not using platform client here also to list resources from the platform ?

Reply:
Since we were already using axios for most of the HTTPs requests, I decided to stick with it.

But I also think it would be cleaner to use the platform-client.